docs/get-started/sumo-logic-ui-new.md

```diff
@@ -0,0 +1,215 @@
+
+<img src={useBaseUrl('/img/get-started/admin-config.png')} alt="admin-config.png" width="350"/>
+
+### Manage your personal account preferences
+
+You can manage your personal account settings from the **Preferences** page. These settings apply only to your account. Changes you make to your preferences take effect the next time you sign in, not during the current session.
+
+To manage your personal Sumo account preferences:
+
+1. From the top nav bar, click the person icon, and then from the dropdown, select **Preferences**.<br/><img src={useBaseUrl('img/get-started/acct-pref.png')} alt="img/get-started/acct-pref.png" width="350"/>
+1. In the Preferences page, you can modify the following settings:
+    * **My Profile**. Username and password.
+    * **My Access Keys**. Add, edit, and remove access keys.
+    * **My Preferences**. Your account session settings.
+
+For more information, see [Account Preferences and Credentials](account-settings-preferences.md).
+
+## Customize your environment with tabs
+
+If you'd prefer to multitask and keep multiple tab open simultaneously (for example: log search, dashboards, App Catalog, and Preferences), we recommend utilizing your own web browser's tab grouping functionality. By adding Sumo Logic tabs to a tab group, any new tabs opened within the Sumo Logic platform will automatically open in the same tab group.
+
+<img src={useBaseUrl('/img/get-started/tabs.png')} alt="tabs.png" width="500" />
+
+This will also allow you to collapse the tab group to reclaim valuable real estate in your browser's tab bar.
+
+## Mastering everyday tasks
+
+This section provides information on how to perform basic everyday tasks using the Sumo Logic UI.
+
+### Analysts (all users)
+
+* [Launch log searches, metrics visualizations, and Live Tail sessions](#launch-searches-metrics-visualizations-and-live-tail-sessions)
+* [View recent dashboards and searches](#view-recent-dashboards-and-searches)
+* [View Favorites and add dashboards and searches to the list](#view-favorites-and-add-dashboards-and-searches-to-the-list)
+* [Share a dashboard, search, or folder](#share-a-dashboard-search-or-folder)
+* [View content that is shared with you](#view-content-that-is-shared-with-you)
+* [Pin and manage searches](#pin-and-manage-searches)
+* [Manage your personal account preferences](#manage-your-personal-account-preferences)
+* [Get help: Sumo Docs, Community, and more](#get-help-docs-community-and-more)
+
+### Administrators
+
+* [Manage data collection, data settings, and alerts](#admin-managedata-collection-data-settings-and-alerts)
+* [Manage accounts, users, and security](#admin-manage-accounts-users-and-security)
```

Comment:
These links are broken. I presume you wanted to actually link to the sections under "Administrator tasks" below.